packages/core/src/components/datetime/test/default/datetime.e2e.ts

```diff
@@ -0,0 +1,63 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+const componentTestPath = 'src/components/datetime/test/default/index.html';
+
+test.describe('tds-datetime-default', () => {
+  test('renders datetime component correctly', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    /* Check diff on screenshot */
+    await expect(page).toHaveScreenshot({ maxDiffPixels: 0 });
+  });
+
+  test('verifies label, helper text, size, and calendar icon for datetime-local component', async ({
+    page,
+  }) => {
+    await page.goto(componentTestPath);
+
+    // Check for the label text
+    const label = page.locator('.tds-datetime-label');
+    const dateTime = page.locator('tds-datetime');
+    const helperText = page.locator('.tds-datetime-helper .tds-helper');
+    const calendarIcon = page.locator('tds-icon[name="calendar"]');
+
+    await expect(dateTime).toHaveAttribute('size', 'lg');
```

Comment:
Checking attributes is good but it can be false positive, for example, attribute can be there but if CSS is wrong, like instead of 56px there is a value of 48px. Please do check height of element too to confirm proper height. I think "toHaveCSS" can be used there :)

Reply:
True, although we do that quite a lot in our tests :). Checking for `toHaveAttribute()`

Reply:
I know and we would have to improve on that. For example, I think if size is component changes the screenshot will fail first, however, it won't tell us more than that screenshots do not match. If we do check the height, it would be an additional test that failed that can help us to isolate the issue faster.

I think it improves test precision, makes it easier for us and contributors to isolate issues much faster. 
What do you think?

Reply:
Yes, I counted at least 33 instances of the usage toHaveAttribute that we need to have a look at then, some very recently merged. Maybe it is better to add toHaveCSS as a compliment instead? An additional check, maybe that is what you meant?

Reply:
Yeah, exactly, to compliment and really confirm that attribute made an effect on component.